src/panels/config/integrations/dialog-add-integration.ts

```diff
@@ -0,0 +1,660 @@
+        @value-changed=${this._filterChanged}
+        .label=${this.hass.localize(
+          "ui.panel.config.integrations.search_brand"
+        )}
+        @keypress=${this._maybeSubmit}
+      ></search-input>
+      ${integrations
+        ? html`<mwc-list
+            style=${styleMap({
+              width: `${this._width}px`,
+              height: `${this._height}px`,
+            })}
+            class="ha-scrollbar"
+            >${integrations.integrations.length
+              ? integrations.integrations.map((integration) =>
+                  this._renderRow(integration)
+                )
+              : html`
+                  <p>
+                    ${this.hass.localize(
+                      "ui.panel.config.integrations.note_about_integrations"
+                    )}<br />
+                    ${this.hass.localize(
+                      "ui.panel.config.integrations.note_about_website_reference"
+                    )}<a
+                      href=${documentationUrl(
+                        this.hass,
+                        `/integrations/${
+                          this._filter ? `#search/${this._filter}` : ""
+                        }`
+                      )}
+                      target="_blank"
+                      rel="noreferrer"
+                      >${this.hass.localize(
+                        "ui.panel.config.integrations.home_assistant_website"
+                      )}</a
+                    >.
+                  </p>
+                `}
+            ${integrations.helpers.length
+              ? html`
+                  <li divider padded class="divider" role="separator"></li>
+                  ${integrations.helpers.map((integration) =>
+                    this._renderRow(integration, true)
+                  )}
+                `
+              : ""}
+          </mwc-list>`
+        : html`<ha-circular-progress active></ha-circular-progress>`} `;
+  }
+
+  private _renderRow(integration: ListItem, is_helper = false) {
+    const manifest = this._manifests?.[integration.domain];
+    return html`
+      <mwc-list-item
+        graphic="medium"
+        .integration=${integration}
+        @click=${this._integrationPicked}
+        hasMeta
+      >
+        <img
+          slot="graphic"
+          loading="lazy"
+          src=${brandsUrl({
+            domain: integration.domain,
+            type: "icon",
+            useFallback: true,
+            darkOptimized: this.hass.themes?.darkMode,
+          })}
+          referrerpolicy="no-referrer"
+        />
+        <span
+          >${integration.name ||
+          domainToName(this.hass.localize, integration.domain)}
+          ${is_helper ? " (helper)" : ""}</span
+        >
+        <div slot="meta">
+          ${!integration.config_flow &&
+          !integration.integrations &&
+          !integration.iot_standards
+            ? html`<span
+                ><ha-svg-icon .path=${mdiCodeTags}></ha-svg-icon
```

Comment:
I'm not sure about the `mdiCodeTags` icon. I didn't understand what it represent before looking at the tooltip.
It's confusing because it's a double arrow and we have already have a right arrow.
![image](https://user-images.githubusercontent.com/5878303/192752665-01ea9073-7188-4d8a-8143-f42b06440b8e.png)